db/migrate/20221110072627_add_settings_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddSettingsToUsers < ActiveRecord::Migration[6.1]
```

Comment:
Also it might be worth bumping the version number of this migration, I was very confused as to why the `settings` column when testing and rolling back.

Reply:
Ah, I started work on this branch in 2022 💀